packages/bpk-component-popover/src/BpkPopoverPortal.js

```diff
@@ -37,12 +37,12 @@ const getClassName = cssModules(STYLES);
 
 export type Props = {
   ...$Exact<PopoverProps>,
-  target: (() => HTMLElement) | Node,
+  target: (() => null | HTMLElement) | Node,
```

Comment:
Is there a specific reason that this is not `() => ?HTMLElement`?

Reply:
Just my lack of experience with flow :) will change :+1: